src/style.css

```diff
@@ -44,19 +51,17 @@ body {
 }
 
 .is-active {
-  box-sizing: border-box;
-  color: #00acdc;
   margin-left: 0;
```

Comment:
not fixed, you don`t need to set margins on the active tab just to make it on the first tab in a row